open-apparel-registry/open-apparel-registry
# Replace newlines with spaces in country strings

## Overview

The list of countries in `countries.py` does not include newlines. Replacing one or more newline characters with a single space should help match country strings with the correct country code.

Connects #2053 

## Notes

I copied the [countries.py list from @KlausGPaul](https://github.com/open-apparel-registry/open-apparel-registry/issues/2053#issuecomment-1257856990) into the repo with one addition:

`'hong kong sar': 'HK'`

## Testing Instructions

* Import or use the api to add a facility with a multiline country string
* The country should parse correctly

## Checklist
